spec/index.bs

```diff
@@ -3954,7 +3959,9 @@ configuring each of the [=render stages=].
                 Any incomplete primitives are dropped.
 
         1. Rasterize.
-            Issue: fill out the section
+            Rasterization is a hardware processing stage that maps the
+            generated primitives to the actual screen.
+            1. Apply {{GPUPrimitiveState/cullMode}}
```

Comment:
?

Reply:
Yeah, this feels like it got pasted into the wrong spot? Culling happens during primitive assembly. Relatedly, there should probably be a brief mention of primitives being culled due to sitting fully outside the normalized device coordinates.